### PR TITLE
Added qq upload instructions to documentation

### PR DESCRIPTION
**Pull request by GabrielePalazzo**
I added instructions explaining how to access the servers with uploaded reports. It is just a copy-paste from `Reame.md`.

Checklist:
- [ ] Reviewers confirm new code works as expected.
- [ ] Tests are passing.
- [ ] Coverage does not decrease.
- [ ] Documentation is updated.
- [ ] Compatibility with Qibo modules (Please edit this section if the current pull request is not compatible with the following branches).
    - [ ] Qibo: `master`
    - [ ] Qibolab: `main`
    - [ ] Qibolab_platforms_qrc: `main`

### CODE REVIEW

File: doc/source/getting-started/interface.rst [hunk 1]
```diff
@@ -76,3 +76,13 @@ by other users as well.
 .. code-block::
 
     qq upload <output_folder>
+
+
+Uploading reports to server
+"""""""""""""""""""""""""""
```
Comment (andrea-pasquale) [inline doc/source/getting-started/interface.rst:82]: I think that the doc looks fine also without this
```suggestion
```

**Review by andrea-pasquale — COMMENTED** [review]
Thanks @GabrielePalazzo.
Since you are already editing the documentation could you please add some documentation on how to add a `tag` when using qq upload?

**Review by AleCandido — APPROVED** [review]
Thanks @GabrielePalazzo for this contribution. To me, it looks fine both with the section title and without (so, in case of doubts, follow @andrea-pasquale advice).

File: doc/source/getting-started/interface.rst (hunk 1 above)
Comment (GabrielePalazzo) [reply doc/source/getting-started/interface.rst]: Ok, I removed it.

File: doc/source/getting-started/interface.rst [hunk 2]
```diff
@@ -76,3 +76,11 @@ by other users as well.
 .. code-block::
 
     qq upload <output_folder>
+
+
+
+
```
Comment (scarrazza) [inline doc/source/getting-started/interface.rst:82]: Thanks @GabrielePalazzo, looks good. Please remove these empty lines (we should aim for the minimum, so 1 or maximum 2 if really needed). Please include a comment about the `--tag` option so we can merge this PR as soon as possible.

Comment (GabrielePalazzo) [reply doc/source/getting-started/interface.rst]: Removed them. I also added a comment about `--tag`.

**Review by scarrazza — APPROVED** [review]

**Review by andrea-pasquale — APPROVED** [review]

**Comment by AleCandido** [comment]
Congratulations to @GabrielePalazzo for his first contribution 🎉

**Pull request merged**